src/containers/ItemDetailContainer.js

```diff
@@ -20,8 +20,16 @@ export class ItemDetailContainer extends Component {
   };
 
   componentDidMount() {
+    this.displayItem(this.props);
+  }
+
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
Is there an alternative lifecycle method to hook into?  Eventually this will get deprecated.